pt-BR/active_record_migrations.md

```diff
@@ -707,114 +707,118 @@ NOTE: Se você quer adicionar verificadores de *constraints* (restrições) como
 você terá que usar `structure.sql` como método *dump*. Consulte
 [Schema Dumping e Você](#schema-dumping-and-you).
 
-Running Migrations
+Executando as *Migrations*
 ------------------
 
-Rails provides a set of rails commands to run certain sets of migrations.
+O Rails fornece uma série de comandos para executar certos conjuntos de
+*migrations*.
 
-The very first migration related rails command you will use will probably be
-`rails db:migrate`. In its most basic form it just runs the `change` or `up`
-method for all the migrations that have not yet been run. If there are
-no such migrations, it exits. It will run these migrations in order based
-on the date of the migration.
+O primeiro comando rails relacionado à *migration* (migração) que você
+utilizará, provavelmente será `rails db:migrate`. Basicamente, ele executa o
+método `change` ou `up` para todas as *migrations* que ainda não foram
+executadas até o momento. Se não houver nenhuma dessas *migrations*, nada é executado. O
+comando executará as *migrations* com base na ordem da data da *migration*.
 
-Note that running the `db:migrate` command also invokes the `db:schema:dump` command, which
-will update your `db/schema.rb` file to match the structure of your database.
+Observe que ao executar o comando `db:migrate` também é chamado o comando
+`db:schema:dump`, que atualizará seu arquivo `db/schema.rb` para corresponder a
+estrutura do seu banco de dados.
 
-If you specify a target version, Active Record will run the required migrations
-(change, up, down) until it has reached the specified version. The version
-is the numerical prefix on the migration's filename. For example, to migrate
-to version 20080906120000 run:
+Se você especificar uma versão alvo, o *Active Record* executará as *migrations*
+necessárias (*change*, *up*, *down*) até atingir a versão especificada. A versão
+é o prefixo numérico do nome do arquivo da *migration*. Por exemplo, para migrar
+para a versão 20080906120000 execute:
 
 ```bash
 $ rails db:migrate VERSION=20080906120000
 ```
 
-If version 20080906120000 is greater than the current version (i.e., it is
-migrating upwards), this will run the `change` (or `up`) method
-on all migrations up to and
-including 20080906120000, and will not execute any later migrations. If
-migrating downwards, this will run the `down` method on all the migrations
-down to, but not including, 20080906120000.
+Se a versão 20080906120000 for maior que a versão atual (ou seja, está migrando
+para cima), será executado o método *change* (ou *up*) em todas as *migrations*
```

Comment:
Fiquei na dúvida aqui sobre voltar pra marcação de código mas como acima usamos itálico achei bem legal essa continuidade parabéns 😄

Reply:
> Fiquei na dúvida aqui sobre voltar pra marcação de código mas como acima usamos itálico achei bem legal essa continuidade parabéns smile

Fiquei em dúvida também. Será que fica melhor voltar pra código? Pensando aqui agora de novo, acho que sim, talvez fique mais coerente. @HenriqueMorato  @campuscode/guia-rails-team

Reply:
Eu ia comentar isso mas na hora de ler não liguei porque o `up`, `down` e `change` da frase anterior não era código aí achei legal a fluidez que deu. 

Mas de repente podemos marcar tudo então?

Reply:
> Eu ia comentar isso mas na hora de ler não liguei porque o `up`, `down` e `change` da frase anterior não era código aí achei legal a fluidez que deu.
> 
> Mas de repente podemos marcar tudo então?

Boa, concordo! Feito em 6c851b689ab59804ee2ea44dcc7c1e0699de56b3